### PR TITLE
Create pahe.js for pahe.in|ph|me

### PR DESCRIPTION
**Pull request by LoSunny**
Fixes (Links to issues fixed by this PR): 
None

Description:
Translating from https://github.com/FastForwardTeam/FastForward/blob/472428a5d788e7c8aae519850629aebdb018a1cb/src/js/injection_script.js#L763

Test links:
None

Checklist:
- [x] I made sure there are no unnecessary changes in the code*
- [ ] Tested on Chromium- Browser OS
- [ ] Tested on Firefox

The `npm run build` doesn't work on MacOS, which I don't have Linux or Win machine to test on 🥲
And am I making mv3 bypasses correctly?

### CODE REVIEW

File: src/bypasses/pahe.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import BypassDefinition from './BypassDefinition.js';
+
+export default class Lnk extends BypassDefinition {
```
Comment (ghost) [inline src/bypasses/pahe.js]: You should change the class name to match the website name
```suggestion
export default class Pahe extends BypassDefinition {
```

**Comment by ghost** [comment]
You should be running `node .\scripts\build.js all` in the root directory to build it. You need to use forward slashes since you mentioned that you are running MacOS.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by LoSunny** [comment]
> You should be running `node .\scripts\build.js all` in the root directory to build it. You need to use forward slashes since you mentioned that you are running MacOS.

It worked, and the extension was running correctly in both Chrome & Firefox. However I don't have any test link to see if the bypass works or not 😅

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
In my testing, it didn't appear to redirect me to the final destination; can you change it so that it does?

**Comment by LoSunny** [comment]
I'll see what I can do in my free time, and understand what is the use of `bypassClipboard`

**Comment by ghost** [comment]
> I'll see what I can do in my free time, and understand what is the use of `bypassClipboard`

bypassClipboard is meant to hold values for later use, but you need to use a function for FF to navigate to the final destination. I think you have the values already needed for the final destination, but it's just a matter of using a function like safelyNavigate.

**Comment by NotAProton** [comment]
bypassClipboard (which was used to inject custom bypass) has been deprecated. ffclipboard is used to save data temporarily

**Comment by ghost** [comment]
Closing this pr because I've created a bypass for intercelestial, which is the meat of the site.

**Comment by LoSunny** [comment]
Thanks for creating one. I'm on vacation and didn't have time to complete it

**Pull request closed without merging**